README.md

```diff
@@ -28,3 +28,27 @@ force-pushes from anywhere.
 
 Additionally, protected repos require code review, and signing. This
 prevents pushes to master/main without a review.
+
+# Run
+
+Ensure you have a GH PAT with org admin permissions in `./org-token`
+
+
+`make manage` will run the command in dryrun mode, printing changes it would have made
+
+`make manage ARGS=--exec` will actually apply the changes.
+
+
+# Cron Job
+
```

Comment:
Can you mention where it is currently installed? 

Holiday curse may lead to people seeking it on dokku2 or whatever...